components/Buttons/src/MDCButton.m

```diff
@@ -51,6 +51,9 @@
 
 static const NSTimeInterval MDCButtonAnimationDuration = 0.2;
 
+static const CGFloat MDCButtonDefaultCornerRadius = 2.0;
+const UIEdgeInsets MDCButtonDefaultContentEdgeInsets = {8, 16, 8, 16};
```

Comment:
These are 0,0,0,0 on standard UIButtons.
Add a comment / documentation on why we are different.

Reply:
Added a comment.